src/options.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8"/>
+  </head>
+
+  <body>
+    <form>
+      <input type="checkbox" id="bookmarksPermissions"/>
+      <label>Enable Bookmark Menus</label>
```

Comment:
For usability, it is generally recommended to either put the `<input>` inside the `<label>`, or to use `<label for="bookmarksPermission">` . Doing so enables the user to toggle the checkbox more easily, as they can then click on the label insted of trying to aim at the checkbox.